afs/media/js/views/components/workflows/project-report-workflow/download-project-files.js

```diff
@@ -0,0 +1,141 @@
+define([
+    'jquery',
+    'js-cookie',
+    'knockout',
+    'arches',
+    'viewmodels/alert-json',
+    'templates/views/components/workflows/project-report-workflow/download-project-files.htm',
+], function($, Cookies, ko, arches, JsonErrorAlertViewModel, downloadFilesTemplate) {
+    function viewModel(params) {
+        var self = this;
+
+        this.projectValue = params.projectId;
+
+        const observationGraphId = '615b11ee-c457-11e9-910c-a4d18cec433a';
+        const digitalResourcegGraphId = '707cbd78-ca7a-11e9-990b-a4d18cec433a';
+        const fileNodeId = '7c486328-d380-11e9-b88e-a4d18cec433a';
+        const fileStatementContentNodeId = 'ca227726-78ed-11ea-a33b-acde48001122';
+        this.relatedPhysicalThings = ko.observableArray();
+        this.relatedObservations = ko.observableArray();
+        this.message = ko.observable();
+
+        this.ready = ko.computed(() => {
+            return self.relatedObservations().find((observation) => {
+                return !!observation.relatedFiles().find(file => file.selected() == true );
+            });
+        });
+
+        this.formatSize = function(size) {
+            var bytes = size;
+            if(bytes == 0) return '0 Byte';
+            var k = 1024;
+            var dm = 2;
+            var sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
It's not clear from a simple reading what this function does (I think you're making the file size human-readable).  Some of these sizes are unnecessary - I doubt we'd ever have terabyte sized files, much less petabyte level files.  Some additional comments and renaming of the variables in this function would be helpful - as well as making these vars const.

Reply:
It is not used any more so removed.